website/client/js/services/userServices.js

```diff
@@ -103,9 +104,19 @@ angular.module('habitrpg')
 
           return Tasks.getUserTasks();
         })
+        // refresh all tasks but completed todos
         .then(function (response) {
           var tasks = response.data.data;
           syncUserTasks(tasks);
+          return Tasks.getUserTasks(true)
```

Comment:
this way there are 2 calls to Tasks.getUserTasks, one at line 105 and one at 111